src/Components/NavigationBar/NavigationBar.tsx

```diff
@@ -11,7 +11,7 @@ export const NavigationBar: Component = () => {
   const [showDocumentationItems, setShowDocumentationItems] =
     createSignal(false);
   const [showMobileNav, setShowMobileNav] = createSignal(false);
-  const [navs] = navSignal;
+  const navs = navMemo;
```

Comment:
I was wondering: do we need those to be reactive values from Solid CreateSignal? Wouldn't importing an array of routes and then rendering Link elements from them work just fine? Do we have a need of changing those values to later update UI? (setNavs...)

Reply:
I don't really think so. I tried to use memos, just in case we have "dynamic" routes in the future :joy: but yeah, we can just import arrays.

Reply:
alright, then since the mechanism for dynamic routes is in place I'm fine with leaving it for later (dynamic nav items is a sign of progress of the app ;D )